File: node/koa/src/business/service/UserService.js
// @flow

/**
 * @function Service层,用于对Model层进行组装或者执行一些逻辑操作
 */
export default class UserService {

  /**
   * @function 生成随机的UserToken,注意,所有纯函数建议写为静态成员函数
   */
  static generateUserToken() {

    return '123456789';

  }

}

File: java/spring/spring-security-jwt/src/main/java/wx/service/UserService.java
package wx.service;

import wx.model.User;
import wx.model.UserDTO;

import java.util.List;

public interface UserService {

    User save(UserDTO user);
    List<User> findAll();
    void delete(long id);
    User findOne(String username);

    User findById(Long id);
}
